internal/tutils/tutils.go

```diff
@@ -75,30 +75,50 @@ func PostResponseBody(t *testing.T, url string, compose interface{}) (*http.Resp
 }
 
 type dB struct {
-	entries []db.ComposeEntry
+	entries          []db.ComposeEntry
```

Comment:
I don't think entries is used at all right now? So we can just drop that field and use map[string][]db.ComposeEntry

Reply:
Sure!